slib.c

```diff
@@ -36,6 +36,8 @@
 // replace character a with b in a string (input string is altered!)
 int slibreplacechr(char *sin, char a, char b)
 {
+	if (sin == NULL)
+		return 1;
```

Comment:
Please don't mix code and variable declarations. Put this after the line below.

It also feels a bit unrelated to the other hunk, so I wouldn't mind seeing it split out into its own commit.

Reply:
Sure, changed it.

Reply:
Sure.

Reply:
Sure, in fact i squashed the changes related to the slibreplacechr function into one introducing it just the way it should be.